javascript/tables/src/guessDataTypeByColumn.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import { DataType } from '@datashaper/schema'
+
+import { guessDataType } from './guessDataType.js'
+
+export function guessDataTypeByColumn(columnValues: string[]): DataType {
```

Comment:
technically this doesn't care if it is a column and could be `guessDataTypeFromValues(values: string[])`